src/galileo_protect/invocation.py

```diff
@@ -76,6 +78,7 @@ async def ainvoke(
         # Set the read timeout to the maximum of the timeout plus the timeout margin.
         read_timeout=timeout + TIMEOUT_MARGIN,
     )
+    logger.debug("Protect invocation completed.")
```

Comment:
Do you want to print timing here?

Reply:
We print the timing as a part of the logger's formatting: https://github.com/rungalileo/core/blob/4705252dc96f46dd0cf983a60ecdde593c4d6983/src/galileo_core/helpers/logger.py#L8

We do debug logging everywhere where we make API requests, so this is just making sure we do it here, too.